### PR TITLE
Update names to no longer use term 'research mode'

### PR DESCRIPTION
**Pull request by klssmith**
The tests in preview and staging no longer run in research mode, so we want to remove all references to research mode from the repo to avoid confusion.

Step one was to add new variable names to notifications-credentials here https://github.com/alphagov/notifications-credentials/pull/170 while not removing the original names yet.

Step two, this commit, now uses these new names.

Step three will be to remove the old names from notifications-credentials.

Old environment variable name | New environment variable name
--- | ---
`NOTIFY_RESEARCH_SERVICE_ID` | `FUNCTIONAL_TESTS_SERVICE_ID`
`NOTIFY_RESEARCH_SERVICE_NAME` | `FUNCTIONAL_TESTS_SERVICE_NAME`
`NOTIFY_RESEARCH_MODE_EMAIL` | `FUNCTIONAL_TESTS_SERVICE_EMAIL`
`NOTIFY_RESEARCH_MODE_EMAIL_PASSWORD` | `FUNCTIONAL_TESTS_SERVICE_EMAIL_PASSWORD`
`NOTIFY_RESEARCH_SERVICE_API_KEY` | `FUNCTIONAL_TESTS_SERVICE_API_KEY`
`NOTIFY_RESEARCH_SERVICE_API_TEST_KEY` | `FUNCTIONAL_TESTS_SERVICE_API_TEST_KEY`
`NOTIFY_RESEARCH_SERVICE_EMAIL_AUTH_ACCOUNT` | `FUNCTIONAL_TESTS_SERVICE_EMAIL_AUTH_ACCOUNT`
`NOTIFY_RESEARCH_EMAIL_REPLY_TO` | `FUNCTIONAL_TESTS_SERVICE_EMAIL_REPLY_TO`
`NOTIFY_RESEARCH_EMAIL_REPLY_TO_2` | `FUNCTIONAL_TESTS_SERVICE_EMAIL_REPLY_TO_2`
`NOTIFY_RESEARCH_EMAIL_REPLY_TO_3` | `FUNCTIONAL_TESTS_SERVICE_EMAIL_REPLY_TO_3`
`NOTIFY_RESEARCH_SERVICE_INBOUND_NUMBER` | `FUNCTIONAL_TESTS_SERVICE_INBOUND_NUMBER`
`NOTIFY_RESEARCH_ORGANISATION_ID` | `FUNCTIONAL_TESTS_ORGANISATION_ID`

### CODE REVIEW

**Comment by tombye** [comment]
Does `environment_local.sh` need updating too?

**Comment by klssmith** [comment]
Yep, everyone will need to change their local environment files once this is merged I'm afraid (I'll put something in Slack when that happens to let people know)

**Comment by klssmith** [comment]
@tombye Yep, thanks, I've done that now and squashed the change into the last commit (I didn't even know that file existed!)

File: environment_local.sh [hunk 1]
```diff
@@ -6,18 +6,18 @@ export TEST_NUMBER=07700900001
...
-
-export NOTIFY_RESEARCH_SERVICE_ID='34b725f0-1f47-49bc-a9f5-aa2a84587c53'
-export NOTIFY_RESEARCH_SERVICE_NAME='Functional Tests'
-export NOTIFY_RESEARCH_ORGANISATION_ID='e6e6ce48-f634-4ebf-af7b-c70fdf16cbd5'
-export NOTIFY_RESEARCH_SERVICE_API_KEY='functional_tests_research_live_key-34b725f0-1f47-49bc-a9f5-aa2a84587c53-846a8975-9e37-4530-ba2b-6ed53819b78d'
-export NOTIFY_RESEARCH_SERVICE_API_TEST_KEY='functional_tests_research_test_key-34b725f0-1f47-49bc-a9f5-aa2a84587c53-ffaf2dec-e7da-4fbe-bcc7-ce6ad51e3b5e'
-
-export NOTIFY_RESEARCH_EMAIL_REPLY_TO='notify-tests-local+reply-to@digital.cabinet-office.gov.uk'
-export NOTIFY_RESEARCH_SERVICE_INBOUND_NUMBER=07700900500
+export FUNCTIONAL_TESTS_SERVICE_EMAIL='notify-tests-local+admin_tests@digital.cabinet-office.gov.uk'
```
Comment (servingUpAces) [inline environment_local.sh:9]: should we update this to a real email address now (or in another PR)? We could something like notify-tests-preview+local@digital.cabinet-office.gov.uk

Comment (klssmith) [reply environment_local.sh:9]: I think updating it here makes sense - I didn't realise that `notify-tests-preview@digital.cabinet-office.gov.uk` was a real email address that could be used to send messages to for testing.

Comment (klssmith) [reply environment_local.sh]: Now updated in the `environment_local.sh` and `db_setup_fixtures.sql` files in this commit https://github.com/alphagov/notifications-functional-tests/pull/332/commits/9d88e4cb738cf0187735d6fa9552ae0575327367

**Pull request merged**